src/Ayaka.MultiTenancy.Abstractions/DependencyInjection/ITenantManagementBuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Raphael Strotz. All rights reserved.
+
+namespace Ayaka.MultiTenancy.DependencyInjection;
+
+using Microsoft.Extensions.DependencyInjection;
+
+/// <summary>
+///     Provides functionality to configure tenant management services.
+/// </summary>
+public interface ITenantManagementBuilder
+{
+    /// <summary>
+    ///     Gets the <see cref="IServiceCollection"/> where tenant management services are configured.
+    /// </summary>
+    IServiceCollection Services { get; }
```

Comment:
Unsure whether this should be `IMultiTenancyBuilder `(as with the `RequestTenancyBuilder`)